pkgs/applications/graphics/processing/default.nix

```diff
@@ -23,7 +23,7 @@ stdenv.mkDerivation rec {
     ${xmlstarlet}/bin/xmlstarlet ed --inplace -P -d '//get[@src="http://download.processing.org/reference.zip"]' build/build.xml
     install -D -m0444 ${fetchurl {
                           url    = "http://download.processing.org/reference.zip";
-                          sha256 = "198bpk8mzns6w5h0zdf50wr6iv7sgdi6v7jznj5rbsnpgyilxz35";
+                          sha256 = "19gj6gvszssg3czac5fcn9yg06739k0i9idc4k37k7wp5yqzs510";
```

Comment:
That's not a reproducible URL and will break again :(. There's also already an upstream issue about this. Maybe use archive.org in the meanwhile?:

```
url = "https://web.archive.org/web/20200406132357/https://download.processing.org/reference.zip";
sha256 = "093hc7kc9wfxqgf5dzfmfp68pbsy8x647cj0a25vgjm1swi61zbi";
```

Reply:
If you change it, you could also add a comment with along the following lines:

```
# Use archive.org link for reproducibility until the following issue is fixed:
# https://github.com/processing/processing/issues/5711
```